doorstop/core/item.py

```diff
@@ -515,7 +514,9 @@ def edit(self, tool=None, edit_all=True):
             self.tree.vcs.lock(self.path)
         # Edit the whole file in an editor
         if edit_all:
+            self.save()
```

Comment:
What is the purpose of this line please ? I don’t get it. I tried to remove it and saw no differences.

Reply:
This is necessary since I removed the auto_save decorator (which was the trouble maker). Before you start the external editor, you have to save the file, afterwards you have to load it. There is no unit test for this.